### PR TITLE
Add support for protobuf Map types

### PR DESCRIPTION
**Pull request by sangjinhan**
The existing Python API crashes upon use of the protobuf Map type, e.g., `MetadataTest` module has:
```
message MetadataTestArg {
  map<string, int64> read = 1;
  map<string, int64> write = 2;
  map<string, int64> update = 3;
}
```

This PR addresses this crash issue, along with fixes for outdated unit tests.

### CODE REVIEW

File: libbess-python/bess.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 import time
 
 import service_pb2
-import proto_conv
+from protobuf_to_dict import *
```
Comment (changlan) [inline libbess-python/bess.py]: It is generally recommended not to use wildcard import. How about `from protobuf_to_dict import protobuf_to_dict, dict_to_protobuf`?

File: libbess-python/protobuf_to_dict_test.py [hunk 2]
```diff
@@ -1,8 +1,7 @@
 import unittest
 import bess_msg_pb2 as bess_msg
 import error_pb2 as error_msg
-import proto_conv
-
+from protobuf_to_dict import *
```
Comment (changlan) [inline libbess-python/protobuf_to_dict_test.py]: Same here.

File: libbess-python/bess.py (hunk 1 above)
Comment (sangjinhan) [inline libbess-python/bess.py]: yes my lord. coming soon...

**Comment by changlan** [comment]
Merged, thanks!
image

**Pull request merged**